lib/order.rb

```diff
@@ -0,0 +1,116 @@
+# require_relative './dish.rb'
+require 'csv'
+require 'sms'
+
+
+class Order
+  attr_reader :available_dishes, :view_dishes, :selected_dishes, :order_total
+
+  def initialize
+    @available_dishes = []
+    @selected_dishes = []
+    load_dishes
+  end
+
+  def load_dishes
+    CSV.foreach("available_dishes.csv") do |line|
+       dish, price, item_number = line
+       #"{imported_menu = []}"
+       @available_dishes << { item_number: item_number, dish: dish, price: price }
+    end
+  end
+
+  def view_menu
+    puts "------------Available Dishes------------"
+      @available_dishes.each do |menu|
+      puts "Item # #{menu[:item_number]} #{menu[:dish]}  Price: £ #{menu[:price]}"
+    end
+  end
+
+  def select_items
+    puts "Please select your desired dishes based on Item number from the menu below"
+    puts "To proceed to your total, please press 'Enter' twice" "\n\n"
+    puts "------------Available Dishes------------"
+    @available_dishes.each do |menu|
+      puts "Item  #{menu[:item_number]}: #{menu[:dish]}  Price: £ #{menu[:price]}"
+      end
+      selection = STDIN.gets.chomp
+      while !selection.empty? do
+        selection = selection.to_i
+        @selected_dishes << @available_dishes[selection-1]
+        selection = STDIN.gets.chomp
+      end
+
+  end
+
+  def order_sum
+    #TODO This is ugly and I would love to find another/better way to get the price to be total-able
```

Comment:
There is an example of a quite tidy approach here, that uses an array of items rather than a hash, that you might find interesting: https://github.com/makersacademy/takeaway-challenge/blob/main/docs/review.md#use-reduce-to-aggregate-over-a-collection-eg-when-calculating-the-total

This also ties into my previous comment about potentially storing Dishes / Items as an object instead.